src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/[[page]]/+page.svelte

```diff
@@ -103,6 +134,21 @@
         </div>
     </div>
 
+    <InputSelectSearch
+        id="related"
+        label="Related Collection"
+        bind:search={selectSearch}
+        bind:value={selectValue}
+        required
+        placeholder="Search by ID"
+        options={filteredOptions}
+        let:option={o}>
+        <div class="u-flex u-gap-16">
+            <span style:color="hsl(var(--color-neutral-70))">{o.label}</span>
```

Comment:
Btw @TGlide I think you can find some already stylized examples at the bottom of the page https://ui-kitchensink-git-style-level-9-appwrite.vercel.app/user.html :)

Reply:
Added custom styling when value is selected, thanks!